README.rst

```diff
@@ -432,6 +432,14 @@ shows the different configuration options available:
         <or>
         platform: linux/arm64/v8 # an apple m1 architecture
 
+        # For Multi-platform images please specify a list of platforms to build for
```

Comment:
How about this instead?

```
To build multi-platform images, add each platform to be built to this list and buildrunner will use docker buildx
to build and tag a single image containing all architectures specified. Note that buildx may be configured to
build some platforms with emulation and therefore builds may take longer with this option specified.
```

Reply:
I made a minor change to that.

> buildrunner will use docker buildx to build and provide a single tag containing all architectures specified.

docker buildx will actually create multiple images, one for each architecture, and then combine them into a single tag.